ui/pages/settings/settings-tab/settings-tab.container.js

```diff
@@ -18,11 +18,12 @@ const mapStateToProps = (state, ownProps) => {
   } = state;
   const {
     currentCurrency,
-    nativeCurrency,
+    providerConfig: { ticker: nativeCurrency },
```

Comment:
after changes on the controller , `nativeCurrency` is removed from `state.metamask`.
it was replaced everywhere with the `ticker` value got from `providerConfig` ( e.g: `getNativeCurrency` function ) but it seems to have been forgotten to replace it at this level too.
and this is why we have this bug